frontend/src/components/common/Resource/Resource.tsx

```diff
@@ -667,7 +667,7 @@ export function OwnedPodsSection(props: OwnedPodsSectionProps) {
 
   const queryData = {
     namespace: resource.kind === 'Namespace' ? resource.metadata.name : undefined,
-    labelSelector: labelSelectorToQuery(resource?.spec?.selector),
+    labelSelector: resource?.spec?.selector ? labelSelectorToQuery(resource?.spec?.selector) : '',
```

Comment:
can we move the logic of returning a empty string in case of no selector is provided into the ```labelSelectorToQuery``` function itself ???

Reply:
I thought about that, but this implies accepting `LabelSelector | undefined` or similar (and then do the validation access || undefined), and in this case I think we should keep it expecting a valid labelSelector and filter out on the caller's side.